README.md

```diff
@@ -1,8 +1,9 @@
 # Gherkin lint
-[![Travis](https://img.shields.io/travis/vsiakka/gherkin-lint.svg?maxAge=2592000)](https://travis-ci.org/vsiakka/gherkin-lint/)
-[![David](https://img.shields.io/david/vsiakka/gherkin-lint.svg?maxAge=2592000)](https://david-dm.org/vsiakka/gherkin-lint)
-[![David](https://img.shields.io/david/dev/vsiakka/gherkin-lint.svg?maxAge=2592000)](https://david-dm.org/vsiakka/gherkin-lint#info=devDependencies&view=table)
+[![Travis](https://img.shields.io/travis/JuanMaRuiz/gherkin-lint.svg?maxAge=2592000)](https://travis-ci.org/JuanMaRuiz/gherkin-lint/)
```

Comment:
I don't think that if i merge this PR the readme will get automatically updated. Do you mind changing these back?

Reply:
I can remove it.